### DISCUSSION TITLE
Improving the fugitive workflow

### DISCUSSION THREAD

**Discussion by PorgJuice** [LazyVim/LazyVim#2638] [Q&A]
Hello! I'm trying to setup a git workflow with LazyGit as my client and vim-fugitive enabled. I'd like to have a nice way of managing conflicts, but the defaults aren't very comfortable. Here's a look at what I've got so far:

![image](https://github.com/LazyVim/LazyVim/assets/6478756/9617ca1e-67a1-4874-b492-ca9620a4b7dc)

Each of the buffers is shown as a tab, the colors could be tweaked to more clearly highlight which diff the section you're looking at is from, I'd like to hide $BASE entirely, and there's conflicts between `:diffput/get` and the delete commands. Here's my current gitconfig:

```
[diff]
  tool = nvimdiff
[difftool]
  prompt = false
[difftool "nvimdiff"]
  cmd = nvim -c Gvdiffsplit! "$MERGED"
[merge]
  tool = nvimdiff
  conflictstyle = diff3
[mergetool]
  prompt = false
[mergetool "nvimdiff"]
  cmd = nvim -f -c Gvdiffsplit! "$MERGED"
```

I'd love any help regarding the above issues, but more generally I'm interested in how people are handling their git workflow with LazyVim. For years I've been using the command line and while I'm not interested in any GUI apps for this, I'd like an open, free, and portable terminal interface. To this end, I'm soliciting opinions and examples. Please, show me what's possible!

**Comment by dusty-phillips** [comment]
It’s not vim-specific, but I primarily use https://graphite.dev/ with the toggle term key binding. I stage right from the editor using the keybindings lazyvim sets up by default using git-signs: `space g h s`, `space g s`, `[h`, `]h` etc. For conflicts, I’ve never found the side by side views terribly helpful, so I just use https://github.com/akinsho/git-conflict.nvim.
